cli/server.go

```diff
@@ -26,11 +31,24 @@ var ServerCmd = cli.Command{
 }
 
 // startEmmyServer configures and starts the gRPC server at the desired port
-func startEmmyServer(port int, certPath, keyPath string) error {
-	srv, err := server.NewProtocolServer(certPath, keyPath)
+func startEmmyServer(port int, certPath, keyPath, logFilePath, logLevel string) error {
+	var err error
+	var logger log.Logger
+
+	if logFilePath == "" {
+		logger, err = log.NewStdoutLogger("server", logLevel, log.FORMAT_LONG)
+	} else {
+		logger, err = log.NewStdoutLogger("server", logLevel, log.FORMAT_LONG_COLORLESS)
```

Comment:
I think `NewStdoutFileLogger` should be used here. At the moment `emmy server start --logfile ./burek.txt` does not log to the file given.